dynamics-nav/Server-Virtual-Table.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Server Virtual Table"
+description: The Server table contains the following fields, Server Name and My Server. This topic contains a link to additional information on how to create a page to view a virtual table.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.